Source/Extensions/Blazorise.DataGrid/DataGrid.razor

```diff
@@ -11,6 +11,13 @@
         @paginationFragment
     }
     <Form>
+
+        @if ( IsCellEdit && ( CommandColumn is null || !CommandColumn.SaveCommandAllowed ) )
+        {
+            //Disable the default form behaviour where pressing enter submits the form and posts the page which is not expected in typical SPA fashion.
+            <Button Type="ButtonType.Submit" Disabled Display="Display.None"></Button>

```

Comment:
I think you should add `PreventDefaultOnSubmit` on a Button here.

Reply:
Same behaviour, the button is not supposed to be triggered anyway.